rotkehlchen/chain/evm/decoding/aura_finance/decoder.py

```diff
@@ -117,9 +121,15 @@ def _decode_lock_aura_bridged(self, context: DecoderContext) -> DecodingOutput:
         return DEFAULT_DECODING_OUTPUT
 
     def _decode_reward_claims(self, context: DecoderContext) -> DecodingOutput:
-        """Decodes Aura Finance reward claiming events."""
+        """Decodes Aura Finance reward claiming events.
+
+        It handles two types of reward transactions: 'getReward' and 'claimRewards'.
+
+        `getReward` transactions are to only claim AURA and BAL rewards.
+        `claimRewards` transactions handle other reward tokens (including AURA and BAL).
+        """
         if (
-            context.tx_log.topics[0] != REWARD_PAID_TOPIC and
+            (is_get_reward := context.transaction.input_data[:4] == GET_REWARD_4BYTE) is False and
             context.transaction.to_address != self.claim_zap_address
         ):
```

Comment:
so that particular decoding rule only runs for the specified methods i.e. getReward or claim rewards.

updated the conditional to be more readable and easy to follow.